README.md

```diff
@@ -34,6 +39,11 @@ To install ipywidgets from git, you will need [npm](https://www.npmjs.com/).
 
         pip install -v -e .
 
+2. Install the Jupyter Widgets nbextension
```

Comment:
`npm install` should invoke `npm run build` because it's set as the `prepublish` script.  This necessity is a funky side effect of using `file:/` package URIs.  It will go away when we break into repos and reference by npm name.